stack.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from linked_list import LinkedList
+
+
+class Stack():
+
+    def __init__(self, iterable=()):
+        self.other = LinkedList()
+        self.other.__init__(iterable)
+
+    def __repr__(self):
+        return self.other.__repr__()
```

Comment:
you shouldn't ever call a special method like `__repr__` directly.  Instead it would be better to do this:

```python
    return repr(self.other)
```